src/scalarstats.jl

```diff
@@ -234,15 +234,23 @@ sem{T<:Real}(a::AbstractArray{T}) = sqrt(var(a) / length(a))
 
 # Median absolute deviation
 """
-    mad(v, center=median(x); constant=1.4826)
+    mad(v)
 
-Compute the median absolute deviation of `v`, optionally specifying a
-precomputed median `center`. `constant` provides a scaling factor that
-defaults to 1.4826 for consistent estimation of the standard deviation
-of a normal distribution.
+Compute the median absolute deviation of `v`.
 """
-mad{T<:Real}(v::AbstractArray{T}, args...;arg...) = mad!(copy(v), args...;arg...)
-mad{T<:Real}(v::Range{T}, args...;arg...) = mad!([v;], args...;arg...)
+function mad{T<:Real}(v::AbstractArray{T})
+    m = median(v)
```

Comment:
> Don't compute this here. Instead call `mad!` without extra arguments.

Don't I need the median so that I can construct `S` to pass it to `mad!`?

> We might want to consider exporting that function.

Which, `StatsBase.mad!` or `LinAlg.copy_oftype`? Exporting the latter sounds like a good idea.